src/command/default/ServerInfo.ts

```diff
@@ -0,0 +1,39 @@
+/***
+ *      _   _      _
+ *     | \ | |    | |
+ *     |  \| | ___| |_ _ __ _____  __
+ *     | . ` |/ _ \ __| '__/ _ \ \/ /
+ *     | |\  |  __/ |_| | |  __/>  <
+ *     |_| \_|\___|\__|_|  \___/_/\_\
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ */
+import Command from '@/command/Command';
+import CommandSender from '@/command/senders/CommandSender';
+import CommandManager from '../CommandManager';
+import Server from '@/server/Server';
+
+class ServerInfoCommand extends Command {
+    public prepare(): void {
+        this.permissions = [];
+        this.permissions.push('netrex.command.stop');
```

Comment:
No point in pushing when you can simply set the array here.

Reply:
k